UI/data/locale/en-US.ini

```diff
@@ -1217,15 +1218,21 @@ YouTube.Actions.Error.NoBroadcastCreated="Broadcast creation error '%1'.<br/>A d
 YouTube.Actions.Error.NoStreamCreated="No stream created. Please relink your account."
 YouTube.Actions.Error.YouTubeApi="YouTube API Error. Please see the log file for more information."
 
+YouTube.Actions.EventsLoading="Loading list of events..."
 YouTube.Actions.EventCreated.Title="Event Created"
 YouTube.Actions.EventCreated.Text="Event successfully created."
 
 YouTube.Actions.ChooseEvent="Choose an Event"
 YouTube.Actions.Stream="Stream"
-YouTube.Actions.Stream.ScheduledFor="scheduled for"
+YouTube.Actions.Stream.ScheduledFor="Scheduled for"
```

Comment:
Could you add a variable for the date? -> "Scheduled for %1"
Some language have a different word order, making this string currently hard to translate.

Reply:
Yes I can.